.travis.yml

```diff
@@ -10,16 +20,15 @@ branches:
     - /^[0-9]+\.[0-9]+$/
 rvm:
   - 2.2.0
+
 before_script:
-  - "cp config/database.yml.$DB config/database.yml"
-  - "bundle exec rake db:setup RAILS_ENV=test"
-  - "sh -e /etc/init.d/xvfb start"
-script: "bundle exec rake travis"
+  - cp config/database.yml.$DB config/database.yml
+  - bundle exec rake db:setup RAILS_ENV=test
+  #- nohup bash -c "webdriver-manager start 2>&1 &"
+  # start rails s for cuckes to hit 
+
+#script: "bundle exec rake travis"
+script: cd lineman && 
+
 addons:
   postgresql: "9.4"
-  code_climate:
-    repo_token:
-      secure: "RQcqTqhx9hhcWO97ib8s8I8m/ufPbBuKRfEEh4RHNIsBEMpqekaPNHtK+T2ym2pfbCpD+L+0bZ+nMlSCrYz0gklfNnYJlPH4/g1f0SX2OEdq2eUCi9xlRObi9ayXsz7Ujtk14mncNGx+e02TnAtEjtFBT8uyvxn8Ql4djvOTsCg="
```

Comment:
ok cool. however is it fine to leave the token in the source code? Is there
another place we can put it maybe?

On Thu, 28 May 2015 at 20:41 James Kiesel <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/loomio/loomio/pull/2200#discussion_r31213406>:
>
> >  addons:
> >    postgresql: "9.4"
> > -  code_climate:
> > -    repo_token:
> > -      secure: "RQcqTqhx9hhcWO97ib8s8I8m/ufPbBuKRfEEh4RHNIsBEMpqekaPNHtK+T2ym2pfbCpD+L+0bZ+nMlSCrYz0gklfNnYJlPH4/g1f0SX2OEdq2eUCi9xlRObi9ayXsz7Ujtk14mncNGx+e02TnAtEjtFBT8uyvxn8Ql4djvOTsCg="
>
> I still use codeclimate occasionally, and it shows up as a badge on our
> README, so I don't think we want to get rid of it just yet.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2200/files#r31213406>.
>